File: The Core/lineUp.cs
int lineUp(string commands) {
    bool matching = true;
    int counter = 0;
    
    foreach(char c in commands){
        
        if((c == 'L' || c == 'R') && matching){
            matching = false;
        }
        else if ((c == 'L' || c == 'R') && !matching){
            matching = true;
            counter++;
        }    
        else if (c == 'A'){
            if(matching) counter++;
        }      
    }
    
    return counter;
}

File: The Core/Loop Tunnel/28. Lineup/lineUp.py
def lineUp(commands):
    matching = True
    counter = 0
    
    for x in list(commands):
        if (x == 'L' or x == 'R') and matching == True:
            matching = False
        elif (x == 'L' or x == 'R') and matching == False:
            matching = True
            counter += 1
        elif (x == 'A'):
            if matching == True:
                counter += 1
    
    return counter
